other/field-types/range.md

```diff
@@ -0,0 +1,26 @@
+# `range`
+
+A `range` field provides a [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) interface for tuning integers. Values are stored as floats.
```

Comment:
Wait, what? Tuning integers? Stored as floats? I'm lost. @abea do you know what this means? I recognize it may be old language.

Reply:
OK looking at the options below, I think this might be better language:

"provides an interface for selecting a numeric value, often represented in the browser as a slider. The step option may be used along with min and max, if desired, to effectively limit the results to integers."

Reply:
Sounds right. "Tuning integers" sounds like something out of the spec. The link to MDN might still be helpful, but Tom's description looks clearer.

Reply:
Fixed so that I can understand it now. :)